modules/virt-cnv-virt-alerts.adoc

```diff
@@ -0,0 +1,487 @@
+[id="VirtOperatorDown_{context}"]
+== VirtOperatorDown alert
+
+.Description
+
+This alert occurs when no virt-operator pod is in the `Running` state in the past 10 minutes. The virt-operator deployment has a default replica of two pods.
+
+.Reason
+
+The virt-operator is the first Kubernetes Operator active in an OpenShift cluster. Its primary responsibilities are:
+
+* Installation
+* Live-update
+* Live-upgrade of a cluster
+* Monitoring the lifecycle of top-level controllers such as virt-controller, virt-handler, and virt-launcher
+* Managing the reconciliation of top-level controllers
+
+In addition, the virt-operator is responsible for cluster-wide tasks such as certificate rotation and some infrastructure management.
+
+[NOTE]
+====
+The virt-operator is not directly responsible for virtual machines in the cluster. The virt-operator's unavailability does not affect the custom workloads.
+====
+
+This alert indicates a failure at the cluster level. Critical cluster-wide management functionalities such as certification rotation, upgrade, and reconciliation of controllers are temporarily unavailable.
+
+.Troubleshoot
+
+. Check the status of the virt-operator deployment to learn more information. These commands provide the associated events and show if there are any specific issues:
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+
+oc -n $NAMESPACE get deploy virt-operator -o yaml
+
+oc -n $NAMESPACE describe deploy virt-operator
+----
+
+. Check the status of the virt-operator pods for further information.
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get kubevirt -A -o custom-columns="":.metadata.namespace)"
+
+oc get pods -n $NAMESPACE -l=kubevirt.io=virt-operator
+----
+
+. Check if there are issues with the nodes for control plane and masters, such as if they are in a `NotReady` state.
++
+[source,terminal]
+----
+oc get nodes
+----
+
+.Resolution
+
+There are several reasons for virt-operator pods being down. Identify the root cause and take appropriate action.
+
+//VirtOperatorRESTErrorsBurst Alert
+[id="VirtOperatorRESTErrorsBurst_{context}"]
+== VirtOperatorRESTErrorsBurst alert
+
+.Description
+
+More than 80% of the REST calls failed in virt-operator in the last five minutes.
+
+.Reason
+
+Virt-operator lost the connection to the API server. Cluster-level actions such as upgrading and controller reconciliation do not function. There is no effect to customer workloads such as VMs and VMIs.
+
+.Troubleshoot
+
+There are two common error types associated with virt-operator REST call failure:
+
+* The API server overloads, causing timeouts. Check the API server metrics and details, such as response times and overall calls.
+
+* The virt-operator pod cannot reach the API server. A common cause is a network connectivity problem, such as DNS issues on the node. Check the virt-operator logs to verify that the pod can connect to the API server at all.
```

Comment:
"A common cause ..... on the node." --> "Common causes are network connectivity problems and DNS issues on the node."